### ISSUE TITLE
[Firefox] Interfering with Dashlane addon

### ISSUE THREAD

**Issue by sudharsannr** [EFForg/privacybadger#1589]
I'm starting to use Privacy Badger and suddenly the Dashlane addon is inactive.  Whenever the Privacy Badger addon is disabled, Dashlane addon becomes active. Contacted the support with Dashlane and they verified that the localhost/127.0.0.1 address is being blocked with Privacy Badger. Added localhost to whitelist and still seems to interfere with Dashlane addon. 

Using Firefox 55.0.2 (32-bit), Windows 10.

**Comment by ghostwords** [comment]
Does Privacy Badger report anything found or blocked on a site without any third-party domains, for example `http://example.com/`?

**Comment by sudharsannr** [comment]
It doesn't report anything blocked on _http://example.com_

**Comment by ghostwords** [comment]
@cowlicks Could you take a quick look into what's going on here and whether there is a workaround?

**Comment by cowlicks** [comment]
@ghostwords sure

**Comment by cowlicks** [comment]
Hmm actually, I'll have to test this later when I have access to a windows or mac machine

**Comment by ghostwords** [comment]
Oh, is there something OS-specific Dashlane requires, besides the Firefox extension?

**Comment by sudharsannr** [comment]
I don't think so. I have a windows application installed.  Here are the steps to reproduce AFAIK:

1. Register an account (free). 
2. Install firefox addon that links to https://www.dashlane.com/firefox-install
3. After the setup is done and account is linked on the firefox addon, you could see a green light on the addon. 
4. Close and re-open firefox. 
5. Green light is turned off. 
6. When privacy badger is disabled, this has to be linked.

**Comment by cowlicks** [comment]
It seems that I need to install the app natively to register.

**Comment by sudharsannr** [comment]
@cowlicks  You are right. Requires an application to be installed. Windows/MacOS only.
